server/controllers/report.js

```diff
@@ -756,6 +756,69 @@ function stock_movement (params){
   return db.exec(requette);
 }
 
+function balance_mensuelle_all (params){
```

Comment:
In SQL, we should stop use the '`' tick marks.  In JavaScript v6 they mean something different.